src/mapclient/view/workflow/workflowwidget.py

```diff
@@ -129,7 +129,8 @@ def applyOptions(self):
         om = self._main_window.model().optionsManager()
         show_step_names = om.getOption(SHOW_STEP_NAMES)
         self._graphicsScene.showStepNames(show_step_names)
-        # self._ui.graphicsView.showStepNames(show_step_names)
+        close_after = om.getOption(CLOSE_AFTER)
+        MessageBox.close_after = close_after * 1000
```

Comment:
Not a fan of this, get the option when it is needed. This option is not one that is "applied".

Reply:
I believe this is related to the comment directly above. Let me know what you think and I can update it.

Reply:
The _close_after_ option is now retrieved upon creation of the MessageBox.